### DISCUSSION TITLE
profiles

### DISCUSSION THREAD

**Discussion by yggdrassilbcn** [seerge/g-helper#2627] [Q&A]
hi. can we make a custom fan and tdp profile  games by game, and applies automatics when open a game? thanks

**Comment by seerge** [comment]
@yggdrassilbcn hello, I was answering this already.

Generally speaking you don't need different profiles for each game. 

But if you want - you can already easily switch between modes [using various hotkeys ](https://github.com/seerge/g-helper?tab=readme-ov-file#%EF%B8%8F-keybindings)

**Reply by yggdrassilbcn** [reply]
just this is what i´m doing.
but i think was great can make profiles for specifics game too.
thanks for the answer
